coursework/product-design/index.md

```diff
@@ -35,11 +35,15 @@ The objective of this assignment is to ideate and design an app that solves a re
 
 For this assignment, you will be designing and building a mockup of your app's user flow. There are a large number of available UX design tools in the industry.
 
-We recommend using [Adobe XD](https://www.adobe.com/sea/products/xd.html), a powerful design tool that is able to convert the designs into interactive mockups too! The interactive mockups are automatically published onto an online link and can be run on mobile devices like the real thing as well. Adobe themselves provide lots of free XD resources and design templates that follow the latest design trends:
+We recommend using [Figma](https://www.figma.com/). Figma is a design tool that is available as a web application. Its most prominent feature is the support for real-time collaboration, allowing team members to work on the prototype at the same time. Figma also has community plugins available for additional functionality, such as Iconify which collates over 50,000 icons for use within your prototypes!
+* <https://www.figma.com/community/plugin/all>
+* <https://www.figma.com/templates/>
+
+Another tool we recommend is [Adobe XD](https://www.adobe.com/sea/products/xd.html), a powerful design tool that is able to convert the designs into interactive mockups too! The interactive mockups are automatically published onto an online link and can be run on mobile devices like the real thing as well. Adobe themselves provide lots of free XD resources and design templates that follow the latest design trends:
```

Comment:
Should we dump XD? I think it may be clearer to have just 1 recommended tool. Plus I feel like Figma is the more popular of the 2 (but I may be wrong)

Reply:
:( Adobe XD ftw

Reply:
I guess we can leave it in for now and see what people pick, can drop it if no one uses :P

Reply:
Actually, Vivian will be doing a session on these design tools. My impression also is that Figma has become v popular.

Reply:
Clarified both options in b90d260b947b21c4204091ed7f1cc3e03384ce09